Common/VR/PPSSPPVR.cpp

```diff
@@ -201,6 +202,58 @@ void UpdateVRInput(bool(*NativeKey)(const KeyInput &key), bool(*NativeTouch)(con
 		}
 	}
 
+	//motion control
+	if (g_Config.bEnableMotions) {
+		for (int j = 0; j < 2; j++) {
+			bool activate;
+			float limit = g_Config.fMotionLength; //length of needed movement in meters
+			XrVector3f axis = {0, 1, 0};
+			float center = ToRadians((float)VR_GetConfig(VR_CONFIG_MENU_YAW));
+			XrQuaternionf orientation = XrQuaternionf_CreateFromVectorAngle(axis, center);
+			XrVector3f position = XrQuaternionf_Rotate(orientation, IN_VRGetPose(j).position);
```

Comment:
When we're checking the position here against the limit, what is it relative too? Some kind of default hand position?

Reply:
It is relative to the head position.

Reply:
Ah I see, that's why the other limit check is * 1.5, your controll base position is normally below the head... Seems to me like an offset first, and then use the same limit, would make more sense than multiplying one of the limits, but this stuff is easily tweakable (doensn't change semantics) so it's fine as-is.